.code-samples.meilisearch.yaml

```diff
@@ -74,6 +74,9 @@ search_get_1: |-
 get_update_1: |-
   curl \
     -X GET 'http://localhost:7700/indexes/movies/updates/1'
+getting_started_get_update: |-
```

Comment:
Could you move this code sample to be next to the other `getting_started` code samples instead of being among the API reference code samples?

Sorry it's a bit nitpicky 😬